packages/notifi-react-card/lib/hooks/useIntercomChat.ts

```diff
@@ -1,32 +1,148 @@
-import { useCallback } from 'react';
+import { ConversationMessagesEntry } from '@notifi-network/notifi-core/dist/models/ConversationMessages';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { ListRange } from 'react-virtuoso';
 
 import { useNotifiClientContext } from '../context';
+import { formatConversationDateTimestamp } from '../utils/datetimeUtils';
+
+export type ChatConversation = FeedEntry &
```

Comment:
Nit: If you are not using the type anywhere else but in the file, you can remove 'export' (because you don't need to export it to anywhere else).

If you have time, it would be great if you could check for the other types that you created for intercom as well :)